presto-main/src/test/java/com/facebook/presto/operator/TestExchangeClient.java

```diff
@@ -122,6 +122,7 @@ public void testHappyPath()
                 true,
                 0.2,
                 new TestingHttpClient(processor, scheduler),
+                (context, header) -> null,
```

Comment:
Maybe having a `TestingDriftExchangeClient` and make it throws ? -- this seems to be the pattern in other place when trying to create something for test.